runner/src/server/plugins/engine/views/components/clientsidefileuploadfield.html

```diff
@@ -261,9 +272,48 @@
         errorSpan.textContent = errorMessage;
     };
 
+    // Show error summery with the error message in js side
+    const handleErrorSummery = (errorMessage) => {
+        const errorSummeryParentContainer = document.querySelector('.govuk-grid-column-two-thirds');
+        const errorSummery = document.querySelector('[data-module="govuk-error-summary"]');
+        if(!errorSummery && errorSummeryParentContainer) {
+            errorSummeryParentContainer.insertAdjacentHTML('afterBegin', `
+                <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
+                    <h2 class="govuk-error-summary__title" id="error-summary-title">
+                        There is a problem
+                    </h2>
+                    <div class="govuk-error-summary__body">
+                        <ul class="govuk-list govuk-error-summary__list">
+                            <li>
+                                <a href="#`+"{{component.model.id}}"+`">`+errorMessage+`</a>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+        `);
+        } else if (errorSummery) {
+            const errorMessageLoc = errorSummery.querySelector(".govuk-error-summary__body > ul > li > a");
+            if (errorMessageLoc) {
+                errorMessageLoc.innerHTML = errorMessage;
+            }
+        }
+    };
+
     const myDropzone = new Dropzone('#dropzoneWrapper', dropzoneConfig);
 
     myDropzone.on('addedfile', function (file) {
+
+        // Intercept save and continue button to prevent submitting the form if files still uploading
+        var saveContinueBtn = document.querySelector('[data-module="govuk-button"]');
```

Comment:
It's rare people use `var` in JavaScript nowdays:

```suggestion
        let saveContinueBtn = document.querySelector('[data-module="govuk-button"]');
```